docs/targets/solana.rst

```diff
@@ -73,6 +73,36 @@ package has `documentation <https://solana-labs.github.io/solana-solidity.js/>`_
 are `some examples <https://solana-labs.github.io/solana-solidity.js/>`_. There is also
 `solang's integration tests <https://github.com/hyperledger/solang/tree/main/integration/solana>`_.
 
+.. _call_anchor:
+
+Calling Anchor Programs from Solidity
+_____________________________________
+
+It is possible to call `Anchor Programs <https://github.com/coral-xyz/anchor>`_
+from Solidity. You first have to generate an interface file from the IDL file using
+the :ref:`metadata_command`. Then, import the Solidity file in your Solidity using the
+``import "...";`` syntax. Say you have an anchor program called `solidity_from_idl` with a
+function `idl_defined_function`, you can call it like so:
+
+.. code-block:: solidity
+
+    import "remoge.sol";
```

Comment:
I know I asked you to change the names. Please, explain what this file is or rename it to `solidity_from_idl.sol`.